### PR TITLE
human bots: do not fight with the ckit

### PR DESCRIPTION
**Pull request by sweet235**
Rather continue building/repairing or go to an armoury to buy a real weapon.

### CODE REVIEW

File: pkg/unvanquished_src.dpkdir/bots/subroutine_fight_or_flight_humans.bt [hunk 1]
```diff
@@ -3,6 +3,7 @@ selector
 	sequence
 	{
 		condition alertedToEnemy
+		condition !haveWeapon( WP_HBUILD )
```
Comment (slipher) [inline pkg/unvanquished_src.dpkdir/bots/subroutine_fight_or_flight_humans.bt:6]: There is `condition haveWeapon( WP_HBUILD )` in the following block so something doesn't make sense here.

Comment (sweet235) [reply pkg/unvanquished_src.dpkdir/bots/subroutine_fight_or_flight_humans.bt:6]: yes

**Comment by slipher** [comment]
LGTM

**Comment by necessarily-equal** [comment]
Without testing, LGTM

**Pull request merged**